WindowsServerDocs/failover-clustering/deploy-two-node-clustered-file-server.md

```diff
@@ -311,33 +311,52 @@ To configure a file server failover cluster, follow the below steps.
 
    ![File Server Type](media/Cluster-File-Server/Cluster-FS-File-Server-Type.png)
 
-8. In the **Client Access Point** window, input the name of the file server you will be using.  Please note that this is not the name of the cluster.  This is for the file share connectivity.  For example, if I want to connect to \\SERVER, the name inputted would be SERVER.
+8. In the **Client Access Point** window, input the name of the file server you are using.  Please note that this is not the name of the cluster.  
+This is for the file share connectivity.  
+For example, if you want to connect to \\\\SERVER, specify the name as SERVER.
```

Comment:
add an extra space, just to be on the safe side (rich diff shows no spacing here):
```suggestion
For example, if you want to connect to  \\\\SERVER, specify the name as SERVER.
```